Makefile

```diff
@@ -35,6 +35,11 @@ htmlcov:
 	go test -v -coverpkg=./... -coverprofile=c.out ./...
 	go tool cover -html ./c.out
 
+.PHONY: golangci-lint
+golangci-lint:
+	$(info INFO: Starting build $@)
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.58.2
+	golangci-lint run --timeout 5m $(pkgs)
 
 lint:
```

Comment:
Maybe we change this to also run golangci-lint.

```
lint: golangci-lint
```

Reply:
I think `golint` does not exist anymore and `golint` can be replaced by `golangci-lint`.

Reply:
Hah, didn't even notice that, yeah.. we can replace `golint` in the `lint` target with `golangci-lint` that makes more sense.

So much has changed in the go ecosystem since Goss was originally written.